app/views/responsiblepeople/experience_training.scala.html

```diff
@@ -43,6 +43,15 @@
     }
 }
 
+@headingFunction = { @businessTypes.map { types =>
+    @if(types.size > 1) {
```

Comment:
same like above

Reply:
if the same function is used twice maybe it would be better to put it into some helper?

Reply:
but I don't mind to leave it as it is without helper :)

Reply:
Done - Came to the agreement not to create helper method as only used twice